public/docs/ts/latest/guide/_data.json

```diff
@@ -151,7 +151,7 @@
 
   "typescript-configuration": {
     "title": "TypeScript Configuration",
-    "intro": "TypeScript configuration for Angular 2 developers"
+    "intro": "This file contains TypeScript configuration for Angular 2 developers."
```

Comment:
I like the original wording better. (Though, like I said, I don't particularly care whether we use a period or not. We just need to be consistent.)